source/ed/docker_tests/ed_integration_tests.cpp

```diff
@@ -122,4 +124,8 @@ BOOST_FIXTURE_TEST_CASE(fusio_test, ArgsFixture) {
     BOOST_CHECK(navitia::contains(headsigns.get_vj_from_headsign("vehiclejourney3"), vj_vec[5]));
 
     BOOST_CHECK_EQUAL(data.pt_data->meta_vjs.size(), 3);
+    for(auto& mvj : data.pt_data->meta_vjs) {
+        BOOST_CHECK_EQUAL(data.pt_data->meta_vjs[mvj->uri], mvj.get());
+        BOOST_CHECK_EQUAL(data.pt_data->meta_vjs[navitia::Idx<nt::MetaVehicleJourney>(mvj->idx)], mvj.get());
```

Comment:
maybe you can typedef that MvjIdx somewhere as it is done in raptor_utils.h for the Idx.